### ISSUE TITLE
"PrestaShop Marketplace in your Back Office" module, Marketplace not loading, reloading automatically

### ISSUE THREAD

**Issue by dheerajwebkul** [PrestaShop/PrestaShop#34013]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

After installing, "[PrestaShop Marketplace in your Back Office](https://addons.prestashop.com/en/administrative-tools/39574-prestashop-marketplace-in-your-back-office.html#overview_description)" module v4.5.3

"Marketplace" tab is just reloading automatically check the video"
https://www.awesomescreenshot.com/video/20945642?key=19f76c5e52f4a19ad0dd21b01b30f681

### Expected behavior

Should load "Marketplace" page properly

### Steps to reproduce

Install, 
https://addons.prestashop.com/en/administrative-tools/39574-prestashop-marketplace-in-your-back-office.html

and check "Marketplace" tab,

### PrestaShop version(s) where the bug happened

8.1.0

### PHP version(s) where the bug happened

8.0.34

### If your bug is related to a module, specify its name and its version

ps_mbo

**Comment by aniszr** [comment]
Hello @dheerajwebkul 

It seems that your issue is not a PrestaShop core bug.

If your module is MBO Official, please contact support team on Addons.

We will close this issue.

Thanks!

**Comment by matks** [comment]
Ping @sowbiba @intraordinaire for information

**Comment by dheerajwebkul** [comment]
@aniszr I have mention module name in issue description, not the prestashop core issue
https://addons.prestashop.com/en/administrative-tools/39574-prestashop-marketplace-in-your-back-office.html#overview_description
it official prestashop module

**Comment by matks** [comment]
@dheerajwebkul I know it's confusing 😅 but this GitHub area is about the open source project PrestaShop https://www.prestashop-project.org/ which is a distinct entity of the company PrestaShop https://prestashop.com/

The MBO module is a service of the PrestaShop company which is out of the scope of this place

You can read more about this here https://build.prestashop-project.org/news/2021/prestashop-beyond-1-7/

Basically the people who can help you with MBO are not here 😉 please reach out to PrestaShop company support team
